### DISCUSSION TITLE
Rust3DS and Github Organizations

### DISCUSSION THREAD

**Discussion by Meziu** [rust3ds/ctru-rs#73] [General]
It’s been months since we joined [Rust3DS](https://github.com/rust3ds), but have yet to gain admin access to actually make changes to the repos. At this point, by also taking into account the obvious lack of willingness of the other members to contribute (which I never expected, but also doesn’t give us much of a reason to stay there), I am pondering the idea of creating a new org with us three (me, @AzureMarker, @ian-h-chamberlain). We could organise all of repos there, and push changes to Rust’s tree to use links pointing to that new org.

Tell me what you think. This is just an idea that got in my mind, but in all honesty, I don’t feel like working in the Rust3DS org is of much help right now.

**Comment by AzureMarker** [comment]
It might be worth trying to reach out via email. There's a neat trick you can do on GitHub commit URLs, just add `.patch`:
https://github.com/HybridEidolon/LFGEnhanced/commit/ec5023f6366773feeb0b409064afaebbba94350d.patch

**Reply by Meziu** [reply]
I’ll try reaching out via email, thank you.

**Reply by Meziu** [reply]
Oh, well, Eidolon did respond, and actually passed me the organisation’s ownership! I will work to update all the work in the org later.
